src/test/java/de/tum/in/www1/artemis/util/DatabaseUtilService.java

```diff
@@ -71,8 +71,8 @@
     public void resetDatabase() {
         conflictRepo.deleteAll();
         conflictingResultRepo.deleteAll();
-        feedbackRepo.deleteAll();
         resultRepo.deleteAll();
+        feedbackRepo.deleteAll();
```

Comment:
I think this is a weird issue - if we delete the feedbacks first, an exception is raised that a feedback with for a result can't be found. It could be the case that something in spring boot / jpa changed about retrieving relationships.
When I switch the order as in this commit, the issue is gone.

`org.springframework.orm.jpa.JpaObjectRetrievalFailureException: Unable to find de.tum.in.www1.artemis.domain.Feedback with id 1; nested exception is javax.persistence.EntityNotFoundException: Unable to find de.tum.in.www1.artemis.domain.Feedback with id 1`

Reply:
I think that the now 2nd line is not really necessary, have a look at the annotations in Result.java:96. It basically states that all feedback items related to a result will be deleted, when the result is deleted (CascadeType.ALL). We also use orphanRemoval = true so a feedback items can basically not exist without the result 'parent'. 

Maybe this has worked in the previous version of Hibernate and now they enforce it.